packages/react/error-boundary/src/useErrorBoundary.en.md

```diff
@@ -0,0 +1,16 @@
+# useErrorBoundary
+
+Hook used to deliver the error to the nearest `ErrorBoundary`.
```

Comment:
I think it would be better to add more description why `useErrorBoundary` is needed.

```markdown
React's `ErrorBoundary` component only catches an error occurred on render or in `useEffect` callback.

So `useErrorBoundary` returns a function to deliver any error occurred in other place to `ErrorBoundary`.
```

Reply:
thank you!
[fix: add docs about why use useErrorBoundary](https://github.com/toss/slash/pull/102/commits/918b0fa67283a34a0054585826dca43361e13a0c)
[918b0fa](https://github.com/toss/slash/pull/102/commits/918b0fa67283a34a0054585826dca43361e13a0c)